### PR TITLE
Misha/ob-365_erat_sieve

### PR DESCRIPTION
**Pull request by krassovki**

### CODE REVIEW

File: Misha/OB-365_erat_sieve/utest_sieve_of_erat.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    def test_erat_sieve_with_minimal_parameter(self):
+        primes = sieve_of_eratosthenes(2)
+        expected_result = [2]
+        self.assertEqual(expected_result, primes)
+
+    def test_erat_sieve_with_non_type_object(self):
+        with self.assertRaises(TypeError) as context:
+            self.assertRaises(TypeError, sieve_of_eratosthenes(None))
+        the_exception = str(context.exception)
```
Comment (glock3) [inline Misha/OB-365_erat_sieve/utest_sieve_of_erat.py]: Change the_exception to exception_message

File: Misha/OB-365_erat_sieve/utest_sieve_of_erat.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        self.assertEqual(expected_result, primes)
+
+    def test_erat_sieve_with_minimal_parameter(self):
+        primes = sieve_of_eratosthenes(2)
+        expected_result = [2]
+        self.assertEqual(expected_result, primes)
+
+    def test_erat_sieve_with_non_type_object(self):
+        with self.assertRaises(TypeError) as context:
+            self.assertRaises(TypeError, sieve_of_eratosthenes(None))
```
Comment (glock3) [inline Misha/OB-365_erat_sieve/utest_sieve_of_erat.py]: You have double check here. Remove one

Comment (glock3) [reply Misha/OB-365_erat_sieve/utest_sieve_of_erat.py]: Same for other occurrence

File: Misha/OB-365_erat_sieve/utest_sieve_of_erat.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import unittest
+from main import sieve_of_eratosthenes
+
+
+class EratSieveTest(unittest.TestCase):
```
Comment (glock3) [inline Misha/OB-365_erat_sieve/utest_sieve_of_erat.py]: Add two more tests
INput:
199
and
100

File: Misha/OB-365_erat_sieve/utest_sieve_of_erat.py (hunk 1 above)
Comment (krassovki) [reply Misha/OB-365_erat_sieve/utest_sieve_of_erat.py]: corrected

File: Misha/OB-365_erat_sieve/utest_sieve_of_erat.py (hunk 3 above)
Comment (krassovki) [reply Misha/OB-365_erat_sieve/utest_sieve_of_erat.py]: added

File: Misha/OB-365_erat_sieve/utest_sieve_of_erat.py (hunk 2 above)
Comment (krassovki) [reply Misha/OB-365_erat_sieve/utest_sieve_of_erat.py]: with self.assertRaises(SomeException) as cm:
                   do_something()
               the_exception = cm.exception
               self.assertEqual(the_exception.error_code, 3)

its an example from ctrl+clisk (case.py)
and i saw same examples in the internet. it is checking, if same error was raised

File: Misha/OB-365_erat_sieve/utest_sieve_of_erat.py (hunk 3 above)
Comment (krassovki) [reply Misha/OB-365_erat_sieve/utest_sieve_of_erat.py]: it works correctly, its the same test as with 5 and 6 values

**Pull request merged**